### PR TITLE
Results: add initial set of results

### PR DESCRIPTION
**Pull request by miltondp**
This PR adds initial sets of results, although the section is not finished yet. The idea to have a glimpse of where things are going and correct now if necessary.

### CODE REVIEW

File: content/04.results.md [hunk 1]
```diff
@@ -1,29 +1,118 @@
...
-> These manuscripts describe new advances, summarize existing literature, or argue for changes in the status quo.
-> Writing and revising manuscripts can be a time-consuming process.
-> Large language models are bringing new capabilities to many areas of knowledge work.
-> We integrated the use of large language models into the Manubot publishing ecosystem.
-> Users of Manubot can run a workflow, which will trigger a series of queries to OpenAI's language models, produce revisions, and create a timestamped set of suggested revisions.
-> Given the amount of time that researchers put into crafting prose, we expect this advance to radically transform the type of knowledge work that academics perform.
-
-After suggested revisions, abstract had a Flesh-Kincaid readability score of X and a grade level of Y and read as follows:
-> NEW_VERSION
+# Evaluating the AI-based revision workflow {#sec:results}
```
Comment (cgreene) [inline content/04.results.md]:
```suggestion
# Observations of AI-based revisions {#sec:results}
```

File: content/04.results.md [hunk 2]
```diff
@@ -1,29 +1,118 @@
...
+
+## Performance of language models
+
+We found that Davinci models, as expected, were superior than the Curie model for all manuscripts.
+The Curie model is described as "very capable", and it is faster and less expensive than Davinci models.
+However, as shown in the PRs generated using this model (titled `GPT (text-curie-001) used to revise manuscript`), the model was not able to produce acceptable revisions for any of the manuscripts.
+Most of its suggestions were not coherent with the original text in any of the sections.
+
+
+Among Davinci models, we found that for `text-davinci-edit-001` (edits endpoint), the quality of the revisions was far inferior than `text-davinci-003` (completion endpoint).
```
Comment (cgreene) [inline content/04.results.md]:
```suggestion
Among Davinci models, we found that for `text-davinci-edit-001` (edits endpoint), the quality of the revisions was subjectively inferior to `text-davinci-003` (completion endpoint).
```
Softened given the limited evaluation.

File: content/04.results.md (hunk 1 above)
Comment (cgreene) [reply content/04.results.md]: I'm not sure this is quite the right thing - essentially you're describing case studies - maybe we should use that language? I like where this is going - it's a bit of observations from the earliest days of this type of tool... but maybe we need a different term like case studies & acknowledge that the review is subjective.

**Review by cgreene — APPROVED** [review]
I think it's good to keep moving this forward, but let's be careful with the language that we're using to make sure reviewers understand what we intend to convey.

File: content/04.results.md (hunk 1 above)
Comment (miltondp) [reply content/04.results.md]: Perfect! I agree.

**Pull request merged**